src/pretix/plugins/checkinlists/exporters.py

```diff
@@ -229,6 +230,8 @@ def _get_queryset(self, cl, form_data, prefetch=True):
             )
         elif sort == 'code':
             qs = qs.order_by(*o, 'order__code')
+        elif sort == 'order_datetime':
+            qs = qs.order_by(*o, 'order__datetime')
```

Comment:
Just read the client’s request, that they want the last orders at the top. So 

```suggestion
            qs = qs.order_by(*o, '-order__datetime')
```